qiskit/unroll/_jsonbackend.py

```diff
@@ -144,9 +144,9 @@ def u(self, arg, qubit, nested_scope=None):
             qubit_indices = [self._qubit_order_internal.get(qubit)]
             self.circuit['operations'].append({
                 'name': "U",
-                'params': [arg[0].real(nested_scope),
-                           arg[1].real(nested_scope),
-                           arg[2].real(nested_scope)],
+                'params': [arg[0].sym(nested_scope),
```

Comment:
Actually I have a doubt about whether these should remain floats or become syms until we verify backend behavior. Perhaps related to the example @adjs gave in the conversation for this PR?